pkg/deploy/deployment_che.go

```diff
@@ -197,6 +179,30 @@ func NewCheDeployment(cr *orgv1.CheCluster, cheImage string, cheTag string, cmRe
 			},
 		},
 	}
+
+	// configure readiness probe if debug isn't set
+	cheDebug := util.GetValue(cr.Spec.Server.CheDebug, DefaultCheDebug)
+	if cheDebug == "false" {
```

Comment:
Maybe :
if cheDebug != true 
?

Since the field is a string, and thus could be empty (which means false) ?